jobs/jenkins_jobs_pr.groovy

```diff
@@ -0,0 +1,67 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = 'jenkins-jobs-pr'
+
+job(name) {
+  description """
+    <ol>
+      <li>Watches the <a href="https://github.com/deis/jenkins-jobs">jenkins-jobs</a> repo for a PR commit</li>
+      <li>And runs tests against changes</li>
+    </ol>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/jenkins-jobs")
+        credentials('597819a0-b0b9-4974-a79b-3a5c2322606d')
+      }
+      refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+      branch('${sha1}')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    githubPush()
+
+    pullRequest {
+      admin('deis-admin')
+      cron('H/5 * * * *')
+      useGitHubHooks()
+      triggerPhrase('OK to test')
```

Comment:
@mboersma there must be other default trigger phrases, perhaps built into the plugin (though an initial cursory search doesn't seem to show [any](https://github.com/janinko/ghprb/search?utf8=%E2%9C%93&q=triggerPhrase)... fwiw we've used `OK to test` in our jobs since we started converting them to DSL.